build.sbt

```
@@ -43,7 +43,16 @@ def scalacCompilerSettings(scalaVer: String) = if (scalaVer.equals(scala210)) {
   scalacExtraParams
 }
 
-val commonSettings = javaCompilerSettings ++ Seq(
+val headerSettings = Seq(
+  headerLicense := {
+    val currentYear = java.time.Year.now(java.time.Clock.systemUTC).getValue
+    Some(HeaderLicense.Custom(
+      s"Copyright (C) 2009-$currentYear Lightbend Inc. <https://www.lightbend.com>"
+    ))
+  }
```

Could we avoid adding the empty line headers, please?

 ```suggestion
  },
  headerEmptyLine := false,
```